resources/views/layout/main_layout.blade.php

```diff
@@ -39,6 +39,7 @@
             @yield('registerCustomer_content')
             @yield('customerLists_content')
             @yield('newTransactions_content')
+            @yield('addUser_content')
```

Comment:
It would be maintainable if we only use one @yield

we can use @yield('content') instead for all contents